### PR TITLE
Convert to openAPI version 3.0.1 and add missing endpoints and parameters

### PR DESCRIPTION
**Pull request by jorilindell**
- Update image filtering parameters and add endpoint to delete images
- Add endpoints to list and retrieve organization classes
- Add endpoints to list and retrieve data sources
- Add missing filtering params to keywords list endpoint and add post, put and delete methods
- Add missing filtering params to keyword sets list endpoint and add post, put and delete methods
- Add missing filtering params to places list endpoint and add post, put and delete methods
- Add missing filtering params to organizations list endpoint and add post, put and delete methods
- Add missing properties to place, organization, keyword, keyword set and event schemas and reorder properties to match api response
- Add event fields which external user is using to event schema

### CODE REVIEW

File: linked-events.swagger.yaml [hunk 1]
```diff
@@ -1,1749 +1,2912 @@
...
+          type: boolean
+          description: Whether this event has been deleted in the original data source
+        maximum_attendee_capacity:
+          type: integer
+          description: Maximum number of people allowed to enrol for the course
+          example: 100
+        minimum_attendee_capacity:
+          type: integer
+          description: Minimum number of people required to enrol for the course
+          example: 10
```
Comment (voneiden) [inline linked-events.swagger.yaml:2288]: https://github.com/City-of-Helsinki/linkedevents/commit/3e5148314009110510750840c7d22c25b12a2705

added

maximum_attendee_capacity_gte
minimum_attendee_capacity_gte
maximum_attendee_capacity_lte
minimum_attendee_capacity_lte


Also is this field usage restricted to the type "course" or should these read just generically "event"?

**Review by voneiden — APPROVED** [review]
Superficially reviewed, schema passes swagger validation.

File: linked-events.swagger.yaml (hunk 1 above)
Comment (jorilindell) [reply linked-events.swagger.yaml:2288]: Added instructions to filter by attendee capacity in https://github.com/City-of-Helsinki/api-linked-events/pull/13/commits/1621362a824ca7c2adaf1acf0b6d37ac42518dc0

**Review by voneiden — APPROVED** [review]
Yep yep, onwards!

**Pull request merged**